src/schema/rules/sidecars/derivatives/functional_derivatives.yaml

```
@@ -0,0 +1,54 @@
+---
+BandpassFilterOpt:
+  selectors:
+  - dataset.DatasetType == "derivative"
+  - '!(["alff", "falff"].includes(suffix))'
+  fields:
+    BandpassFilter: optional
+BandpassFilterReq:
+  selectors:
+  - dataset.DatasetType == "derivative"
+  - '["alff", "falff"].includes(suffix)'
+  fields:
+    BandpassFilter: required
```

I think we can just do this a a single table:

```suggestion
BandpassFilter:
  selectors:
  - dataset.DatasetType == "derivative"
  - '["alff", "falff"].includes(suffix)'
  fields:
    BandpassFilter:
      level: required
      level_addendum: for suffixes `alff` and `falff`
```